cmd/validator/validator.go

```diff
@@ -161,6 +170,15 @@ func isFlagSet(flagName string) bool {
 	return isSet
 }
 
+func setFlagFromEnvIfNotSet(flagName string, envVar string) {
+	if !isFlagSet(flagName) {
+		envVarValue, isEnvVarSet := os.LookupEnv(envVar)
+		if isEnvVarSet {
+			flag.Set(flagName, envVarValue)
+		}
+	}
+}
```

Comment:
I would avoid a function wide `if`, and use early return

```suggestion
func setFlagFromEnvIfNotSet(flagName string, envVar string) {
	if isFlagSet(flagName) {
		return
	}
	envVarValue, isEnvVarSet := os.LookupEnv(envVar)
	if isEnvVarSet {
		flag.Set(flagName, envVarValue)
	}
}
```